src/index.ts

```diff
@@ -217,12 +217,14 @@ export async function exportKey(key: CryptoKey): Promise<string> {
  * @param password - The password to use to generate key.
  * @param salt - The salt string to use in key derivation.
  * @param exportable - Should the derived key be exportable.
+ * @param iterations - The number of iterations to use in key derivation.
  * @returns A CryptoKey for encryption and decryption.
  */
 export async function keyFromPassword(
   password: string,
   salt: string,
   exportable = false,
+  iterations = 900_000,
```

Comment:
Numeric separators were added in ES2021, but the compilation target for this package is ES2017. Is it possible that this might break at runtime?

Reply:
Also TIL, apparently BigInt is vulnerable to timing attacks: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#cryptography

Reply:
On second thought, the TypeScript compiler will probably transpile this correctly. But it might be worth it to double check.

Reply:
Looks like TypeScript handles this dynamically according to the compilation target (numeric separators are only preserved in es2021+/esnext and discarded otherwise):
https://github.com/microsoft/TypeScript/pull/41435/files#diff-49e0e0aa62fb4f804b58e25e49f10dd494c69b5bd9f1524bbaa2f512448c5a2c. 

Consider this resolved!

Reply:
@MajorLift thanks a lot for taking the time to look into this!